README.md

```diff
@@ -260,7 +260,7 @@ In Shipping Go you will learn how to:
  - Scale your deployment in a cost-effective way
  - Deliver a culture of continuous improvement
 
-### 2024 - [Learning Go: An Idiomatic Approach to Real-World Go Programming, 2nd Edition](https://amzn.eu/d/0ExgXvJ)
+### 2024 - [Learning Go: An Idiomatic Approach to Real-World Go Programming, 2nd Edition](https://www.amazon.de/-/en/Jon-Bodner/dp/1098139291)
```

Comment:
It's probably best to use the publisher (O'Reilly) page instead of Amazon for a specific country.

Reply:
Are there any updates to the summary in the second edition?

Reply:
The old amazon link was basically same except the id which I felt is standard enforced by the  maintainers. Shall I update it with Oreilly link?

Reply:
🤷🏼‍♂️